src/frontend/screens/Audio/useAudioPlayback.ts

```diff
@@ -0,0 +1,85 @@
+import {Audio, AVPlaybackStatus, AVPlaybackStatusSuccess} from 'expo-av';
+import {useCallback, useEffect, useState, useRef} from 'react';
+import {Sound} from 'expo-av/build/Audio/Sound';
+
+export const useAudioPlayback = (recordingUri: string) => {
+  const recordedSoundRef = useRef<Sound | null>(null);
+  const [isPlaying, setPlaying] = useState(false);
+  const [duration, setDuration] = useState<number>(0);
+  const [currentPosition, setCurrentPosition] = useState(0);
+  const [isReady, setReady] = useState(false);
```

Comment:
I think we can take out isReady completely (or don't make it state). Its not really being used in a meaningful way, and we don't export it